webportal/webapp/src/app/account/loginsync/loginsync.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="container">
+  <mat-card>
+    <div>
+      <form class="usa-form signin" (ngSubmit)="onSubmit()">
+        <fieldset class="usa-fieldset">
+          <legend class="usa-legend">Sign in</legend>
+  
+          <label for="username">Username</label>
+          <input class="usa-input" id="username" name="username" [(ngModel)]="username" type="email" autocapitalize="off" autocorrect="off">
```

Comment:
Can you remove the other email specific types?

Reply:
woops, nice catch. 👍